ci-operator/step-registry/vsphere-acm-virt/cold-migration/vsphere-acm-virt-cold-migration-commands.sh

```
@@ -0,0 +1,375 @@
+# Power off source VM on vSphere before cold migration
+# --------------------------------------------------------------------------
+if [[ -f "${SHARED_DIR}/govc-env.sh" ]]; then
+    # govc-env.sh contains only non-sensitive context; no set +x needed.
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/govc-env.sh"
+
+    # Reload credentials from mount since they are not in govc-env.sh.
+    typeset _wasTracing2=false
+    [[ $- == *x* ]] && _wasTracing2=true
+    set +x
+    [[ -f "${credsDir}/.vsphere_user"     ]] && export GOVC_USERNAME="$(< "${credsDir}/.vsphere_user")"     || \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)